cypress_test/integration_tests/desktop/impress/top_toolbar_spec.js

```diff
@@ -93,12 +93,12 @@ describe('Top toolbar tests.', function() {
 		cy.get('#tb_editbar_item_fonts')
 			.click();
 		
-		desktopHelper.selectFromListbox('Lato Thin');		
+		desktopHelper.selectFromListbox('Liberation Sans');		
```

Comment:
Liberation Sans is the font name, what that text shape already has when we open the test document. So you should not use that here, because this does not actually change the document, so it won't catch that when the feature is broken. You can try 'Liberation Mono' for example, if you don't have better option in your mind.